From 290073d7d4ddaa6d39ad12c90245e795c5482432 Mon Sep 17 00:00:00 2001
From: Arfrever Frehtes Taifersar Arahesis <Arfrever@Apache.Org>
Date: Sat, 29 Jan 2022 22:43:50 +0800
Subject: [PATCH] protobuf-3.18.0: disable no-warning test

Disable no-warning-test which is compiled with -Werror option
and whose only purpose is checking if compilation results in any warnings.

Signed-off-by: Arfrever Frehtes Taifersar Arahesis <Arfrever@Apache.Org>
Signed-off-by: Yixun Lan <dlan@gentoo.org>
---
 src/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index e6b4d0f..1cfd885 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -726,7 +726,7 @@ GOOGLETEST_SRC_DIR=$(srcdir)/../third_party/googletest/googletest
 GOOGLEMOCK_SRC_DIR=$(srcdir)/../third_party/googletest/googlemock
 check_PROGRAMS = protoc protobuf-test protobuf-lazy-descriptor-test \
                  protobuf-lite-test test_plugin protobuf-lite-arena-test \
-                 no-warning-test $(GZCHECKPROGRAMS)
+                 $(GZCHECKPROGRAMS)
 protobuf_test_LDADD = $(PTHREAD_LIBS) libprotobuf.la libprotoc.la \
                       $(GOOGLETEST_BUILD_DIR)/lib/libgtest.la     \
                       $(GOOGLEMOCK_BUILD_DIR)/lib/libgmock.la     \
@@ -915,4 +915,4 @@ nodist_no_warning_test_SOURCES = no_warning_test.cc $(protoc_outputs)
 
 TESTS = protobuf-test protobuf-lazy-descriptor-test protobuf-lite-test \
         google/protobuf/compiler/zip_output_unittest.sh $(GZTESTS)     \
-        protobuf-lite-arena-test no-warning-test
+        protobuf-lite-arena-test
-- 
2.34.1

